resources/js/oauth/new-client.tsx

```diff
@@ -42,19 +44,40 @@ export class NewClient extends React.Component {
         <form autoComplete='off' className='oauth-client-details__content'>
           {this.renderRemainingErrors()}
 
-          {NewClient.inputFields.map((name) => (
-            <label key={name} className='oauth-client-details__group'>
-              <div className='oauth-client-details__label'>{trans(`oauth.client.${name}`)}</div>
-              <ValidatingInput
-                blockName='oauth-client-details'
-                errors={this.errors}
-                name={name}
-                onInput={this.handleInput}
-                type='text'
-                value={this.params[name]}
-              />
-            </label>
-          ))}
+          <label className='oauth-client-details__group'>
+            <div className='oauth-client-details__label'>
+              {trans('oauth.client.name')}
+            </div>
+            <ValidatingInput
+              blockName='oauth-client-details'
+              errors={this.errors}
+              name='name'
+              onChange={this.handleOnChangeName}
+              type='text'
+              value={this.params.name}
+            />
+          </label>
+
+          <label className='oauth-client-details__group'>
+            <div className='oauth-client-details__label'>
+              {trans('oauth.client.redirect')}
+            </div>
+            <TextareaAutosize
```

Comment:
Might be useful to initialize the textarea height to at least the number of callbacks within reason; that said, I feel like there should be a limit on the number of callbacks that can be set on a single application 🤔

Reply:
the min height is already set in the css though?

Reply:
I mean if there's 5 callbacks it should start at 5 rows visible, instead of the current one where it only gets larger after adding more callbacks